api/api.proto

```diff
@@ -4239,6 +4242,21 @@ message SdkCloudBackupInfo {
   map<string, string> metadata = 5;
   // Status indicates the status of the backup
   SdkCloudBackupStatusType status = 6;
+  // cluster indicates if the cloudbackup belongs to current cluster,
+  // with older cluster this value may be unknown
+  SdkCloudBackupClusterType cluster_type = 7;
+  // k8s namespace to which this backup belongs to
+  string namespace = 8; 
+}
+
+// CloudBackup operations types
+enum SdkCloudBackupClusterType {
```

Comment:
Enum is not complient with the SDK. Please see https://github.com/libopenstorage/openstorage/blob/master/docs/dev-sdk.md#enums